dev-python/python-eventlib/python-eventlib-0.2.1-r1.ebuild

```diff
@@ -20,7 +20,11 @@ KEYWORDS="~amd64 ~x86"
 
 RDEPEND="
 	dev-python/greenlet[${PYTHON_USEDEP}]
-	dev-python/twisted-core[${PYTHON_USEDEP}]
+	dev-python/twisted[${PYTHON_USEDEP}]
 "
 
 PATCHES=( "${FILESDIR}/${P}-libressl.patch" )
+
+python_test() {
+        "${PYTHON}" -m unittest discover || die
```

Comment:
`-v`, please.

Reply:
I tried:

>>> Test phase: dev-python/python-msrplib-0.19.1
Usage: python -m unittest [options]

python -m unittest: error: no such option: -v
test_basic (unittest.loader.ModuleImportFailure) ... ERROR

Reply:
Try after `discover`.

Reply:
yes, that was clear, sorry I didn't put the code to make it easier from your side:

```
python_test() {
        pushd test > /dev/null || die
        "${EPYTHON}" -m unittest discover -v || die
        popd > /dev/null || die
}
```

Reply:
Let's focus on getting the missing test-dependencies first, and then I'll try to figure out how to solve the verbosity problem.